hv-rhel5.x/hv/rhel5-hv-driver-install

```diff
@@ -1,12 +1,17 @@
 echo "Building Modules"
-make -C /lib/modules/$(uname -r)/build M=`pwd` clean
+if [ "$KERNEL_VERSION" == "" ]
```

Comment:
Perhaps we should pass the target KERNEL_VERSION as a parameter to this script.

Reply:
I thought about that, but the buildrpms.sh also needs a tweak, and if you don't call make but instead call the buildrpms you have to pass it there, then pass it down to the make. All doable, but then there is pbuild, which I don't have a handle on.  Since it tars the sources, then builds it, you need to have something set up at the pbuild level, to pass to rpmbuild, to pass to make in order for this all to work. Basically, I was trying to be non-invasive to the pbuild process.

Reply:
I'm not too familiar with pBuild. @vyadavmsft can you comment on this?